static/figure/js/views/right_panel_view.js

```diff
@@ -243,6 +243,7 @@
         events: {
             "submit .new-label-form": "handle_new_label",
             "click .dropdown-menu a": "select_dropdown_option",
+            "click .btn-markdown": "show_markdown"
```

Comment:
This isn't working for me, possibly because the ```.show_markdown``` button is not in the ```labels_form_inner_template.html``` that is dynamically added to the ```#labelsTab```. It's in the original index.html.
One option is to simply set this up just once at the start.
E.g. You can add this to the ```$(function(){})``` in ```util.js```:
```
$("#labelsTab .btn-markdown").click(function(){$("#markdownInfoModal").modal('show');});
```

Ah, sorry - just realised that the reason it's not working on eel is that you haven't committed the figure.js concatenated file.

Another thing you need to handle is the conversion of markdown to html in the PDF export script.
Let me know if you need help with that. Got to go...